pkg/plugin/trans.go

```diff
@@ -273,6 +273,37 @@ func (r resourceTransition[T]) handleAutoscalingDisabled() (verdict string) {
 	return verdict
 }
 
+// handleStartMigration updates r.node with changes to clear any buffer and capacityPressure from
+// r.pod.
+//
+// If the pod is the migration source, this method *also* increases the node's PressureAccountedFor
+// to match the pod's resource usage.
+func (r resourceTransition[T]) handleStartMigration(source bool) (verdict string) {
+	// The first bit of this is _mostly_ the same as handleAutoscalingDisabled.
```

Comment:
I'd avoid leaving the difference that isn't covered by "mostly" as an exercise to the reader.

Reply:
Done